src/cli/lint/internalLints/refreshConfigFiles.test.ts

```diff
@@ -252,6 +288,27 @@ The .dockerignore file is out of date. Run \`pnpm exec skuba format\` to update
       expect(writeFile).not.toHaveBeenCalled();
     });
 
+    it('should report ok if local skuba version is out of date, and output nothing', async () => {
+      getDestinationManifest.mockResolvedValue({
+        packageJson: {
+          skuba: {
+            version: '2147483647.0.0',
+          },
+        },
+      } as any);
+
+      setupDestinationFiles({});
+
+      await expect(refreshConfigFiles('format', log)).resolves.toEqual({
+        ok: true,
+        fixable: false,
+      });
+
+      expect(stdout()).toBe('');
+
+      expect(writeFile).not.toHaveBeenCalled();
+    });
```

Comment:
Is it possible for us to tell the user to do a `ni`? Having out of date local packages can cause other issues